build/tar.go

```diff
@@ -0,0 +1,80 @@
+package build
+
+import (
+	"archive/tar"
+	"io"
+	"log"
+	"os"
+	"path"
+	"path/filepath"
+)
+
+func tarDir(destinationFilename, sourceDir string) {
+
```

Comment:
nitpick: Can you remove this extra line?

Reply:
done. Removed extra line below `walkTar` header as well